ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Send.hs

```diff
@@ -289,62 +298,69 @@ createNewReciever uiLangFace@UiLangFace{..} s@Send{..} = do
   addSeparator layout
 
-  QBoxLayout.addWidget recieversLayout recieverWidget
+  QBoxLayout.addWidget receiversLayout receiverWidget
 
   connect_ amountEdit QLineEdit.textChangedSignal $ \_ -> do
     revalidate s
     recalcTotal uiLangFace s
   connect_ addressEdit QLineEdit.textChangedSignal $ \_ -> revalidate s
 
-  let r = Reciever{..}
+  let r = Receiver{..}
 
   connect_ removeButton QAbstractButton.clickedSignal $ \_ -> do
-    removeReciever uiLangFace r s
+    removeReceiver uiLangFace r s
 
-  modifyIORef recievers (r :)
-  QWidget.adjustSize recieversWidget
+  modifyIORef receivers (r :)
+  QWidget.adjustSize receiversWidget
   QWidget.adjustSize send
   revalidate s
 
-removeReciever :: UiLangFace -> Reciever -> Send -> IO ()
-removeReciever uiLangFace@UiLangFace{..} Reciever{..} s@Send{..} = do
-  enabledRecievers <- getEnabledRecievers s
-  QWidget.setEnabled recieverWidget False
-  QWidget.setVisible recieverWidget False
+removeReceiver :: UiLangFace -> Receiver -> Send -> IO ()
+removeReceiver uiLangFace@UiLangFace{..} Receiver{..} s@Send{..} = do
+  enabledReceivers <- getEnabledReceivers s
+  QWidget.setEnabled receiverWidget False
+  QWidget.setVisible receiverWidget False
   revalidate s
   recalcTotal uiLangFace s
-  -- To prevent removing last reciever
-  if length enabledRecievers == 1
+  -- To prevent removing last receiver
+  if length enabledReceivers == 1
     then do
-      createNewReciever uiLangFace s
+      createNewReceiver uiLangFace s
     else do
-      QWidget.adjustSize recieversWidget
+      QWidget.adjustSize receiversWidget
       QWidget.adjustSize send
 
-getEnabledRecievers :: Send -> IO [Reciever]
-getEnabledRecievers Send{..} = do
-  r <- readIORef recievers
-  filterM (\Reciever{..} -> QWidget.isEnabled recieverWidget) r
+getEnabledReceivers :: Send -> IO [Receiver]
+getEnabledReceivers Send{..} = do
+  r <- readIORef receivers
+  filterM (\Receiver{..} -> QWidget.isEnabled receiverWidget) r
 
 recalcTotal :: UiLangFace -> Send -> IO ()
 recalcTotal UiLangFace{..} s@Send{..} = do
-  enabledRecievers <- getEnabledRecievers s
-  textAmounts <- mapM (\Reciever{..} -> fromString <$> QLineEdit.text amountEdit) enabledRecievers
-  let amounts = map Unsafe.fromJust (filter isJust (map (\a -> normalize <$> readMaybe a) textAmounts))
-  langPutUiCommand (UiCalcTotal amounts) >>= \case
-    Left err -> void $ QMessageBox.critical send ("Error" :: String) $ toString err
-    Right _ -> pass
-
-changeTotalAmount :: Send -> Text -> IO ()
-changeTotalAmount Send{..} amount = do
+  enabledReceivers <- getEnabledReceivers s
+  textAmounts <- mapM (fmap fromString . QLineEdit.text . amountEdit) enabledReceivers
+  let amounts = catMaybes (filter isJust (map (fmap normalize . readMaybe) textAmounts))
+  -- To prevent summarizing amounts, that typed improperly
+  if length enabledReceivers == length amounts
+    then do
+      QWidget.setStyleSheet totalAmount ("QLabel { color : #34343e }" :: String)
```

Comment:
I'd prefer if you used `styleRole` property or something like that and put those colors into `stylesheet.qss`. Also please consult with @a-rukin which gray color fits to our colorscheme.

Reply:
AFAIU, to use `QWidget.setProperty` in runtime will require `QStyle`, which is not presented in qtah
https://wiki.qt.io/Dynamic_Properties_and_Stylesheets
Please correct me, if I'm wrong

Reply:
Oh come ooon. That's sad :(